force-app/main/default/lwc/platformDevCertCalculator/platformDevCertCalculator.js

```diff
@@ -0,0 +1,89 @@
+import { LightningElement, track } from 'lwc';
+    const devFundWeight = .23;
+    const processAutoWeight = .30;
+    const userIntWeight = .25;
+    const testDebugWeight = .22;
+
+export default class PlatformDevCertCalculator extends LightningElement {
+
+    devFundamentalScore = 50;
+    processAutoScore = 50;
+    userInterfaceScore = 50;
+    testDebugDeployScore = 50;
+    passingScore = 70; 
+
+    certificationScore = 90;
+    numberOfQuestions = 60;
+
+    showResources = false;
+    showGoodJob = false;
+
+    currentHistoryId = 0;
+
+    @track attemptHistory = [
+        {Id: 1, Score:50},
+        {Id: 2, Score:25},
+        {Id: 3, Score:100},
+    ]
+    
+
+    
+    calculateScore(){
+        let devFundWeightScore = this.devFundamentalScore * devFundWeight;
+        let processAutoWeightScore = this.processAutoScore * processAutoWeight;
+        let userIntWeightScore = this.userInterfaceScore * userIntWeight;
+        let testDebugWeightScore = this.testDebugDeployScore * testDebugWeight;
+        this.certificationScore = devFundWeightScore + processAutoWeightScore + userIntWeightScore + testDebugWeightScore;
+
+        this.showResourcesIfFailed();
+        this.addAttemptHistory(this.certificationScore);
+    }
+
+    handleChange(event){
+        console.log(event.target.value, event.target.name);
+        console.log(event.target.type);
+        console.log(event.target.label);
+
+        const inputName = event.target.name;
+        let value = Number(event.target.value);
```

Comment:
Try adding a validation and toast if the users enters a value less than 0 or greater than 100